singer-io/tap-pendo
# Tdl 14963 removed endpoints dict

# Description of change

- removed the endpoints dictionary
- added an Endpoints Class

# Manual QA steps
 - Check if the endpoint API URL is correct for each stream.
 - Check that the new code generates the same catalog files as before
 - Check that the new code generates the same records with and without the state file as before.

 
# Risks
 - 
 
# Rollback steps
 - revert this branch
